bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -1433,6 +1433,18 @@ private function addWorkflowNode(ArrayNodeDefinition $rootNode)
                                     ->defaultNull()
                                     ->info('Will be applied when the current place is empty.')
                                 ->end()
+                                ->arrayNode('initial_markings')
+                                    ->beforeNormalization()
```

Comment:
add info node with short description & information that this option will work with symfony 4 only ... since symfony 3 does not allow an array as inital place ... see also https://github.com/symfony/workflow/blob/3.4/Definition.php